app/javascript/angular/code/controllers/_fixed_sessions_map_ctrl.js

```diff
@@ -1,19 +1,6 @@
 import _ from "underscore";
-import moment from "moment";
-
 import * as FiltersUtils from "../filtersUtils";
 
-const endOfToday = moment()
-  .utc()
-  .endOf("day")
-  .format("X");
-
-const oneYearAgo = moment()
-  .utc()
-  .startOf("day")
-  .subtract(1, "year")
-  .format("X");
```

Comment:
this was setting static values once instead of every time we needed to refresh. with endOfToday and oneYearAgo the difference may have not been noticeable but with the 1 hour span it was obvious we need functions